### ISSUE TITLE
Semantic headings retrofit  (severity 3)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1601]
This ticket addresses the semantic structure of the pages, without looking at the visual presentation (which should likely be reassessed after these changes are made). The proposed changes are based on observations in exhibits.stanford.edu only.

Proposed changes:

- [x] Make the banner text (currently in `<div class="site-title h2">`) an `<h1>` for all pages
   - Exclude the subtitle from the `<h1>` tag
   - Affects:  All pages
   - Some pages will end up with two `<h1>` headings, but this doesn't appear to be harmful.

- [x] Make sidebar titles (in red text) `<h2>` where they are `<h4>`
  - Affects:  About, Feature
  - Home & Search results are already correct

- [ ] Consider increasing widget headings from `<h3>` to `<h2>`
   - Curators have the ability to structure their page with `<h2>` headings followed by widgets with `<h3>`s. Unfortunately, many exhibits exist without `<h2>` headings, where the curator has just placed a widget or group of widgets on the page. In many cases, the addition of an `<h2>` would be artificial: if there's only one widget, for example, further structure isn't required. 
   - Problem arises if the widget heading level is increased to `<h2>` and the curator DOES add an explicit `<h2>` heading... then the structure is incorrect in a different way (that won't appear in automated audits). A longer term solution might allow an explicit "group these under this heading" action.

- [x] Make hidden headings match the associated menu text
  - I can't see how these customizations are made but I see them in existing exhibits: "Collections" or "Explore" instead of "Browse". It's a bit disorienting for the resulting page to have a different heading. 
  - Affects: Browse categories (index page)

### Home page
- Missing `<h1>`

### Exhibit home
- Missing `<h1>`
- Widget headings are `<h3>`

### Feature
- `<h1>` in the content area (ok)
- `<h4>`s in the sidebar appear before the content `<h1>`

### Browse categories
- hidden `<h1>` "Browse exhibit" 
   - doesn't match menu option or breadcrumb

### Category :+1:
- hidden `<h1>` is category name

### Item :+1:
- `<h1>` is item title
- second `<h1>` for "Mirador viewer"

### Search result :+1:
- hidden `<h1>` is "Search"
- hidden page structure is complete, no missed levels

### About
- `<h1>` in content
   - doesn't match menu option or breadcrumb ... ok?

**Comment by jkeck** [comment]
Related projectblacklight/spotlight#2216 (we should work to get this merged as well)

**Comment by jkeck** [comment]
@jvine for
> Make sidebar titles (in red text) `<h2>` where they are `<h4>`

We have a similar heading structure in the admin dashboards (Configuration / Curation ).  Should these headings also get this update?  The heading structure is a bit different on the dashboards.  I think this is less critical at the moment, and the code is decoupled so I can make the changes independently, but did want to call it out (and perhaps that ends up getting spun off into a separate issue).

**Comment by jkeck** [comment]
For 
> Consider increasing widget headings from `<h3>` to `<h2>`

I'm wondering if this is something we should keep as-is.  We ultimately can't control what users put in their pages and how they structure them (and curator driven content's semantic structure can't really be our responsibility).  Also, in general this may end up causing people lots of grief w/ updating in case they curator has carefully thought out their heading structure (and we would end up breaking that w/ this update).  Holding off for now, but happy to discuss further and be swayed. 😃
